src/Core/TileGeometry.js

```diff
@@ -222,12 +225,10 @@ TileGeometry.prototype.computeBuffers = function computeBuffers(params, builder)
     var iStart = idVertex;
 
     // TODO: WARNING beware skirt's size influences performance
-    // Fix Me: Compute correct the skirt's size : minimize the size without crack between tiles
-    // This size must be take into account the bbox's size
-    // For the moment, I reduce the size to increase performance (pixel shader performance)
-
+    // The size of the skirt is now a ration of the size of the tile.
+    // To be perfect it should depend on the real elevation delta but too heavy to compute
     if (!params.disableSkirt) {
-        const r = 5 * ((20 - params.level) + 10);
+        const r = segmentSize; // Tile size divided by the number of segments
```

Comment:
How using `segmentSize` will be better than any other arbitrary formula?

IMHO the issue you're referencing is caused by the different subdivision level of 2 neighbors, so maybe we should also adjust the subdivision algorithm / SSE to prevent this problem.